### PR TITLE
Fix enum value representation with new lang changes

### PR DESCRIPTION
**Pull request by lnash94**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by lnash94** [comment]
Related discussions https://github.com/ballerina-platform/openapi-tools/pull/950#discussion_r884395865

File: openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java [hunk 1]
```diff
@@ -405,7 +405,7 @@ private Schema mapEnumValues(EnumSymbol enumSymbol) {
         List<ConstantSymbol> enumMembers = enumSymbol.members();
         for (ConstantSymbol enumMember : enumMembers) {
             if (enumMember.typeDescriptor().typeKind() == TypeDescKind.SINGLETON) {
-                enums.add(enumMember.typeDescriptor().signature());
+                enums.add(enumMember.typeDescriptor().signature().replaceAll("\"", ""));
```
Comment (NipunaRanasinghe) [inline openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java:408]: This logic might fail when there are escaped double quotes present within the string literal itself. 
(e.g: `string foo = "Hello \"World\""`). So shall we add a test case for a similar scenario?

**Review by NipunaRanasinghe — CHANGES_REQUESTED** [review]

File: openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java (hunk 1 above)
Comment (lnash94) [reply openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java:408]: Yeah will add the test for it.

**Review by NipunaRanasinghe — COMMENTED** [review]
@lnash94 Shall we also add a test case for a enum value expression scenario? (i.e. string concatenation)

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**